.github/workflows/release.yml

```diff
@@ -39,12 +39,17 @@ jobs:
           printf '::set-output name=version::%s\n' "$(./build/bin/nvim --version | head -n 3 | sed -z 's/\n/%0A/g')"
           printf '::set-output name=release::%s\n' "$(./build/bin/nvim --version | head -n 1)"
           make DESTDIR="$GITHUB_WORKSPACE/build/release/nvim-linux64" install
-          cd "$GITHUB_WORKSPACE/build/release"
-          tar cfz nvim-linux64.tar.gz nvim-linux64
+          cd "$GITHUB_WORKSPACE/build/"
+          cpack -C $NVIM_BUILD_TYPE
```

Comment:
This is cool, have wanted to better leverage cpack for years. Thanks for doing this!